rules/aws_db_instance_invalid_engine.go

```diff
@@ -0,0 +1,80 @@
+package rules
+
+import (
+	"fmt"
+
+	hcl "github.com/hashicorp/hcl/v2"
+	"github.com/terraform-linters/tflint-plugin-sdk/tflint"
+	"github.com/terraform-linters/tflint-ruleset-aws/project"
+)
+
+// AwsDBInstanceInvalidEngineRule checks whether "aws_db_instance" has invalid engine.
+type AwsDBInstanceInvalidEngineRule struct {
+	resource      string
+	attributeName string
+	engines       map[string]bool
+}
+
+// NewAwsDBInstanceInvalidEngineRule returns new rule with default attributes
+func NewAwsDBInstanceInvalidEngineRule() *AwsDBInstanceInvalidEngineRule {
+	return &AwsDBInstanceInvalidEngineRule{
+		resource:      "aws_db_instance",
+		attributeName: "engine",
+		engines: map[string]bool{
+			// https://docs.aws.amazon.com/cli/latest/reference/rds/describe-db-engine-versions.html#options
+			"aurora":            true,
+			"aurora-mysql":      true,
+			"aurora-postgresql": true,
+			"mariadb":           true,
+			"mysql":             true,
+			"oracle-ee":         true,
+			"oracle-se2":        true,
+			"oracle-se1":        true,
+			"oracle-se":         true,
+			"postgres":          true,
+			"sqlserver-ee":      true,
+			"sqlserver-se":      true,
+			"sqlserver-ex":      true,
+			"sqlserver-web":     true,
+		},
+	}
+}
+
+// Name returns the rule name
+func (r *AwsDBInstanceInvalidEngineRule) Name() string {
+	return "aws_db_instance_invalid_type"
+}
+
+// Enabled returns whether the rule is enabled by default
+func (r *AwsDBInstanceInvalidEngineRule) Enabled() bool {
+	return true
+}
+
+// Severity returns the rule severity
+func (r *AwsDBInstanceInvalidEngineRule) Severity() string {
+	return tflint.ERROR
+}
+
+// Link returns the rule reference link
+func (r *AwsDBInstanceInvalidEngineRule) Link() string {
+	return project.ReferenceLink(r.Name())
```

Comment:
This rule doesn't have documentation, so it's better to return an empty string instead.

By the way, the `aws_db_instance_invalid_type` rule also returns a link, but since there is no documentation, it seems that this should also return an empty string... 😓

Reply:
Understand, cool 🆒 
https://github.com/terraform-linters/tflint-ruleset-aws/blob/master/project/main.go#L10
https://github.com/terraform-linters/tflint-ruleset-aws/tree/v0.2.1/docs/rules

I will add a document for the rule 👍

Reply:
Addressed 👍  https://github.com/terraform-linters/tflint-ruleset-aws/pull/61/commits/6a9575c5f3ed4e2b92fb230a5220433e11ece464 
How about this?